Is the docker build broken?

I followed the Augeas [docker/README](https://github.com/hercules-team/augeas/blob/master/docker/README.md) to build the image in my local, and encountered several errors.

Here's my friction report:  

Problem 1: `docker build` fails with `env: ‘bash’: No such file or directory`
- Logs:
```bash
docker build -t augeas -f docker/Dockerfile .
[+] Building 14.7s (9/11)                                                                                              
 => [internal] load build definition from Dockerfile                                                              0.2s
 => => transferring dockerfile: 724B                                                                              0.0s
 => [internal] load .dockerignore                                                                                 0.2s
 => => transferring context: 2B                                                                                   0.0s
 => [internal] load metadata for docker.io/library/alpine:latest                                                  1.9s
 => [internal] load build context                                                                                 0.7s
 => => transferring context: 11.54MB                                                                              0.7s
 => [build 1/4] FROM docker.io/library/alpine@sha256:7580ece7963bfa863801466c0a488f11c86f85d9988051a9f9c68cb27f6  1.0s
 => => resolve docker.io/library/alpine@sha256:7580ece7963bfa863801466c0a488f11c86f85d9988051a9f9c68cb27f6b7872   0.0s
 => => sha256:7580ece7963bfa863801466c0a488f11c86f85d9988051a9f9c68cb27f6b7872 1.64kB / 1.64kB                    0.0s
 => => sha256:9b2a28eb47540823042a2ba401386845089bb7b62a9637d55816132c4c3c36eb 528B / 528B                        0.0s
 => => sha256:d7d3d98c851ff3a95dbcb70ce09d186c9aaf7e25d48d55c0f99aae360aecfd53 1.47kB / 1.47kB                    0.0s
 => => sha256:530afca65e2ea04227630ae746e0c85b2bd1a179379cbf2b6501b49c4cab2ccc 2.80MB / 2.80MB                    0.5s
 => => extracting sha256:530afca65e2ea04227630ae746e0c85b2bd1a179379cbf2b6501b49c4cab2ccc                         0.4s
 => [build 2/4] RUN apk add --no-cache --virtual=.build gcc libc-dev make bison flex readline-dev libxml2-dev g  10.9s
 => [stage-1 2/4] RUN apk add --no-cache libgcc libxml2 readline bash                                             2.0s
 => [build 3/4] COPY . /augeas                                                                                    0.2s
 => ERROR [build 4/4] RUN set -ex ;     cd augeas  && ./autogen.sh --prefix=/opt/augeas  && make  && make instal  0.3s 
------                                                                                                                 
 > [build 4/4] RUN set -ex ;     cd augeas  && ./autogen.sh --prefix=/opt/augeas  && make  && make install:            
#9 0.255 + cd augeas                                                                                                   
#9 0.255 + ./autogen.sh '--prefix=/opt/augeas'                                                                         
#9 0.257 env: ‘bash’: No such file or directory                                                                        
------                                                                                                                 
executor failed running [/bin/sh -c set -ex ;     cd augeas  && ./autogen.sh --prefix=/opt/augeas  && make  && make install]: exit code: 127  
```
- My fix: I added "bash" in the Dockerfile [the alpine install step](https://github.com/hercules-team/augeas/blob/fa315ce5a59b01f0a3f7730a70857f009f5db2d9/docker/Dockerfile#L3)  


Problem 2: #9 `225.1 transform.c:921:5: error: unknown type name 'security_context_t'; did you mean 'security_class_t'?` 
- logs:
```bash
docker build -t augeas -f docker/Dockerfile .
[+] Building 237.1s (9/11)                                                                                             
 => [internal] load build definition from Dockerfile                                                              0.0s
 => => transferring dockerfile: 747B                                                                              0.0s
 => [internal] load .dockerignore                                                                                 0.0s
 => => transferring context: 2B                                                                                   0.0s
 => [internal] load metadata for docker.io/library/alpine:latest                                                  0.0s
 => [internal] load build context                                                                                 0.1s
 => => transferring context: 41.55kB                                                                              0.1s
 => CACHED [build 1/4] FROM docker.io/library/alpine                                                              0.0s
 => CACHED [stage-1 2/4] RUN apk add --no-cache libgcc libxml2 readline bash                                      0.0s
 => [build 2/4] RUN apk update && apk add bash --no-cache --virtual=.build gcc libc-dev make bison flex readlin  11.7s
 => [build 3/4] COPY . /augeas                                                                                    0.2s
 => ERROR [build 4/4] RUN set -ex ;     cd augeas  && ./autogen.sh --prefix=/opt/augeas  && make  && make inst  225.1s 
------                                                                                                                 
 > [build 4/4] RUN set -ex ;     cd augeas  && ./autogen.sh --prefix=/opt/augeas  && make  && make install:            
#9 0.307 + cd augeas                                                                                                   
#9 0.307 + ./autogen.sh '--prefix=/opt/augeas'                                                                         
#9 0.563 libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, 'build/ac-aux'.                                     
#9 0.563 libtoolize: copying file 'build/ac-aux/ltmain.sh'                                                             
#9 0.605 libtoolize: putting macros in 'gnulib/m4'.
#9 0.605 libtoolize: copying file 'gnulib/m4/libtool.m4'
#9 0.613 libtoolize: copying file 'gnulib/m4/ltoptions.m4'
#9 0.622 libtoolize: copying file 'gnulib/m4/ltsugar.m4'
#9 0.634 libtoolize: copying file 'gnulib/m4/ltversion.m4'
#9 0.641 libtoolize: copying file 'gnulib/m4/lt~obsolete.m4'
#9 0.647 libtoolize: Consider adding 'AC_CONFIG_MACRO_DIRS([gnulib/m4])' to configure.ac,
#9 0.647 libtoolize: and rerunning libtoolize and aclocal.
#9 0.651 ./bootstrap: getting gnulib files...
#9 0.676 Submodule 'gnulib' (git://git.sv.gnu.org/gnulib.git) registered for path '.gnulib'
#9 0.700 Cloning into '/augeas/.gnulib'...
#9 112.6 Submodule path '.gnulib': checked out '2f3892304bd432c5ca3f291b3ef7d8a912a85e96'
#9 126.8 Notice from module vasprintf:
#9 126.8   If you are using GNU gettext version 0.16.1 or older, add the following options
#9 126.8   to XGETTEXT_OPTIONS in your po/Makevars:
#9 126.8     --flag=asprintf:2:c-format --flag=vasprintf:2:c-format
#9 128.3 Creating directory ./gnulib/lib
#9 128.3 Creating directory ./gnulib/tests
#9 128.3 Creating directory ./gnulib/lib/glthread
#9 128.3 Creating directory ./gnulib/lib/malloc
#9 137.5 Creating gnulib/lib/Makefile.am
#9 137.5 Creating gnulib/m4/gnulib-cache.m4
#9 140.6 Creating gnulib/m4/gnulib-comp.m4
#9 146.7 Creating gnulib/tests/Makefile.am
#9 146.7 Creating ./build/ac-aux/.gitignore
#9 147.4 Creating ./gnulib/lib/.gitignore
#9 147.4 Creating ./gnulib/lib/glthread/.gitignore
#9 147.5 Creating ./gnulib/lib/malloc/.gitignore
#9 148.2 Creating ./gnulib/m4/.gitignore
#9 149.2 Creating ./gnulib/tests/.gitignore
#9 149.2 Creating ./gnulib/tests/glthread/.gitignore
#9 149.2 Finished.
#9 149.2 
#9 149.2 You may need to add #include directives for the following .h files.
#9 149.4   #include <argz.h>
#9 149.4   #include <ctype.h>
#9 149.4   #include <fnmatch.h>
#9 149.4   #include <getopt.h>
#9 149.4   #include <locale.h>
#9 149.4   #include <regex.h>
#9 149.4   #include <selinux/context.h>
#9 149.4   #include <selinux/label.h>
#9 149.4   #include <selinux/selinux.h>
#9 149.4   #include <stdio.h>
#9 149.4   #include <stdlib.h>
#9 149.4   #include <string.h>
#9 149.4   #include <sys/wait.h>
#9 149.4   #include "safe-alloc.h"
#9 149.9 
#9 149.9 You may need to use the following Makefile variables when linking.
#9 149.9 Use them in <program>_LDADD when linking a program, or
#9 149.9 in <library>_a_LDFLAGS or <library>_la_LDFLAGS when linking a library.
#9 149.9   $(LIBTHREAD)
#9 149.9   $(LIB_CLOCK_GETTIME)
#9 149.9   $(LIB_GETRANDOM)
#9 149.9   $(LIB_HARD_LOCALE)
#9 149.9   $(LIB_MBRTOWC)
#9 149.9   $(LIB_SELINUX)
#9 149.9   $(LIB_SETLOCALE_NULL)
#9 149.9   $(LTLIBINTL) when linking with libtool, $(LIBINTL) otherwise
#9 149.9 
#9 149.9 Don't forget to
#9 149.9   - add "gnulib/lib/Makefile" to AC_CONFIG_FILES in ./configure.ac,
#9 149.9   - add "gnulib/tests/Makefile" to AC_CONFIG_FILES in ./configure.ac,
#9 149.9   - mention "lib" in SUBDIRS in gnulib/Makefile.am,
#9 149.9   - mention "tests" in SUBDIRS in gnulib/Makefile.am,
#9 149.9   - mention "-I gnulib/m4" in ACLOCAL_AMFLAGS in Makefile.am,
#9 149.9   - mention "gnulib/m4/gnulib-cache.m4" in EXTRA_DIST in Makefile.am,
#9 149.9   - invoke gl_EARLY in ./configure.ac, right after AC_PROG_CC,
#9 149.9   - invoke gl_INIT in ./configure.ac.
#9 156.0 configure.ac:71: installing 'build/ac-aux/compile'
#9 156.0 configure.ac:74: installing 'build/ac-aux/config.guess'
#9 156.0 configure.ac:74: installing 'build/ac-aux/config.sub'
#9 156.0 configure.ac:5: installing 'build/ac-aux/install-sh'
#9 156.0 configure.ac:5: installing 'build/ac-aux/missing'
#9 156.3 examples/Makefile.am: installing 'build/ac-aux/depcomp'
#9 156.7 parallel-tests: installing 'build/ac-aux/test-driver'
#9 156.9 configure.ac: installing 'build/ac-aux/ylwrap'
#9 157.0 tests/Makefile.am:306: warning: 'INCLUDES' is the old name for 'AM_CPPFLAGS' (or '*_CPPFLAGS')
#9 157.0 tests/Makefile.am:308: warning: source file '$(top_srcdir)/src/memory.c' is in a subdirectory,
#9 157.0 tests/Makefile.am:308: but option 'subdir-objects' is disabled
#9 157.0 automake: warning: possible forward-incompatibility.
#9 157.0 automake: At least one source file is in a subdirectory, but the 'subdir-objects'
#9 157.0 automake: automake option hasn't been enabled.  For now, the corresponding output
#9 157.0 automake: object file(s) will be placed in the top-level directory.  However, this
#9 157.0 automake: behavior may change in a future Automake major version, with object
#9 157.0 automake: files being placed in the same subdirectory as the corresponding sources.
#9 157.0 automake: You are advised to start using 'subdir-objects' option throughout your
#9 157.0 automake: project, to avoid future incompatibilities.
#9 157.0 tests/Makefile.am:320: warning: source file '$(top_srcdir)/src/memory.c' is in a subdirectory,
#9 157.0 tests/Makefile.am:320: but option 'subdir-objects' is disabled
#9 157.0 tests/Makefile.am:314: warning: source file '$(top_srcdir)/src/memory.c' is in a subdirectory,
#9 157.0 tests/Makefile.am:314: but option 'subdir-objects' is disabled
#9 157.0 tests/Makefile.am:326: warning: source file '$(top_srcdir)/src/memory.c' is in a subdirectory,
#9 157.0 tests/Makefile.am:326: but option 'subdir-objects' is disabled
#9 157.0 tests/Makefile.am:323: warning: source file '$(top_srcdir)/src/memory.c' is in a subdirectory,
#9 157.0 tests/Makefile.am:323: but option 'subdir-objects' is disabled
#9 157.0 tests/Makefile.am:317: warning: source file '$(top_srcdir)/src/memory.c' is in a subdirectory,
#9 157.0 tests/Makefile.am:317: but option 'subdir-objects' is disabled
#9 157.0 tests/Makefile.am:311: warning: source file '$(top_srcdir)/src/memory.c' is in a subdirectory,
#9 157.0 tests/Makefile.am:311: but option 'subdir-objects' is disabled
#9 158.5 configure.ac:4: warning: 'AM_CONFIG_HEADER': this macro is obsolete.
#9 158.5 configure.ac:4: You should use the 'AC_CONFIG_HEADERS' macro instead.
#9 158.5 ./lib/autoconf/general.m4:2434: AC_DIAGNOSE is expanded from...
#9 158.5 aclocal.m4:1089: AM_CONFIG_HEADER is expanded from...
#9 158.5 configure.ac:4: the top level
#9 158.5 configure.ac:59: warning: The macro `AC_HELP_STRING' is obsolete.
#9 158.5 configure.ac:59: You should run autoupdate.
#9 158.5 ./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
#9 158.5 ./lib/autoconf/general.m4:1534: AC_ARG_ENABLE is expanded from...
#9 158.5 configure.ac:59: the top level
#9 158.5 configure.ac:71: warning: The macro `AC_GNU_SOURCE' is obsolete.
#9 158.5 configure.ac:71: You should run autoupdate.
#9 158.5 ./lib/autoconf/specific.m4:312: AC_GNU_SOURCE is expanded from...
#9 158.5 configure.ac:71: the top level
#9 158.5 configure.ac:84: warning: The macro `AC_PROG_LIBTOOL' is obsolete.
#9 158.5 configure.ac:84: You should run autoupdate.
#9 158.5 gnulib/m4/libtool.m4:100: AC_PROG_LIBTOOL is expanded from...
#9 158.5 configure.ac:84: the top level
#9 158.5 configure.ac:86: warning: AC_PROG_LEX without either yywrap or noyywrap is obsolete
#9 158.5 ./lib/autoconf/programs.m4:716: _AC_PROG_LEX is expanded from...
#9 158.5 ./lib/autoconf/programs.m4:709: AC_PROG_LEX is expanded from...
#9 158.5 configure.ac:86: the top level
#9 158.5 configure.ac:88: warning: The macro `AC_HELP_STRING' is obsolete.
#9 158.5 configure.ac:88: You should run autoupdate.
#9 158.5 ./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
#9 158.5 acinclude.m4:10: AUGEAS_COMPILE_WARNINGS is expanded from...
#9 158.5 configure.ac:88: the top level
#9 158.5 configure.ac:88: warning: The macro `AC_TRY_LINK' is obsolete.
#9 158.5 configure.ac:88: You should run autoupdate.
#9 158.5 ./lib/autoconf/general.m4:2920: AC_TRY_LINK is expanded from...
#9 158.5 acinclude.m4:10: AUGEAS_COMPILE_WARNINGS is expanded from...
#9 158.5 configure.ac:88: the top level
#9 158.5 configure.ac:133: warning: AC_OUTPUT should be used without arguments.
#9 158.5 configure.ac:133: You should run autoupdate.
#9 159.1 checking for a BSD-compatible install... /usr/bin/install -c
#9 159.4 checking whether build environment is sane... yes
#9 159.4 checking for a race-free mkdir -p... /bin/mkdir -p
#9 159.4 checking for gawk... no
#9 159.4 checking for mawk... no
#9 159.4 checking for nawk... no
#9 159.4 checking for awk... awk
#9 159.4 checking whether make sets $(MAKE)... yes
#9 159.4 checking whether make supports nested variables... yes
#9 159.5 checking whether make supports nested variables... (cached) yes
#9 159.5 checking for naturaldocs... no
#9 159.5 checking for NaturalDocs... no
#9 159.5 checking whether make supports the include directive... yes (GNU style)
#9 159.5 checking for gcc... gcc
#9 159.5 checking whether the C compiler works... yes
#9 159.7 checking for C compiler default output file name... a.out
#9 159.7 checking for suffix of executables... 
#9 159.7 checking whether we are cross compiling... no
#9 159.8 checking for suffix of object files... o
#9 159.8 checking whether the compiler supports GNU C... yes
#9 159.8 checking whether gcc accepts -g... yes
#9 159.8 checking for gcc option to enable C11 features... none needed
#9 160.0 checking whether gcc understands -c and -o together... yes
#9 160.0 checking whether the compiler is clang... no
#9 160.0 checking for compiler option needed when checking for declarations... none
#9 160.0 checking dependency style of gcc... gcc3
#9 160.1 checking for stdio.h... yes
#9 160.1 checking for stdlib.h... yes
#9 160.1 checking for string.h... yes
#9 160.2 checking for inttypes.h... yes
#9 160.2 checking for stdint.h... yes
#9 160.2 checking for strings.h... yes
#9 160.3 checking for sys/stat.h... yes
#9 160.3 checking for sys/types.h... yes
#9 160.3 checking for unistd.h... yes
#9 160.3 checking for wchar.h... yes
#9 160.4 checking for minix/config.h... no
#9 160.4 checking for sys/param.h... yes
#9 160.5 checking for sys/socket.h... yes
#9 160.5 checking for fnmatch.h... yes
#9 160.5 checking for getopt.h... yes
#9 160.6 checking for sys/cdefs.h... no
#9 160.6 checking for threads.h... yes
#9 160.6 checking for limits.h... yes
#9 160.7 checking for langinfo.h... yes
#9 160.7 checking for xlocale.h... no
#9 160.7 checking for sys/mman.h... yes
#9 160.7 checking for malloc.h... yes
#9 160.8 checking for selinux/selinux.h... no
#9 160.8 checking for sys/time.h... yes
#9 160.8 checking for sys/random.h... yes
#9 160.9 checking for sys/wait.h... yes
#9 160.9 checking for features.h... yes
#9 160.9 checking for crtdefs.h... no
#9 161.0 checking for wctype.h... yes
#9 161.0 checking for arpa/inet.h... yes
#9 161.1 checking for netdb.h... yes
#9 161.1 checking for netinet/in.h... yes
#9 161.1 checking for semaphore.h... yes
#9 161.2 checking for sys/select.h... yes
#9 161.2 checking for pthread.h... yes
#9 161.2 checking for sys/ioctl.h... yes
#9 161.3 checking for sys/uio.h... yes
#9 161.3 checking whether it is safe to define __EXTENSIONS__... yes
#9 161.3 checking whether _XOPEN_SOURCE should be defined... no
#9 161.4 checking for gcc... (cached) gcc
#9 161.4 checking whether the compiler supports GNU C... (cached) yes
#9 161.4 checking whether gcc accepts -g... (cached) yes
#9 161.4 checking for gcc option to enable C11 features... (cached) none needed
#9 161.4 checking whether gcc understands -c and -o together... (cached) yes
#9 161.4 checking whether the compiler is clang... (cached) no
#9 161.4 checking for compiler option needed when checking for declarations... (cached) none
#9 161.4 checking dependency style of gcc... (cached) gcc3
#9 161.4 checking build system type... x86_64-pc-linux-musl
#9 161.5 checking host system type... x86_64-pc-linux-musl
#9 161.5 checking how to run the C preprocessor... gcc -E
#9 161.5 checking for grep that handles long lines and -e... /bin/grep
#9 161.6 checking for egrep... /bin/grep -E
#9 161.6 checking for Minix Amsterdam compiler... no
#9 161.6 checking for ar... ar
#9 161.6 checking for ranlib... ranlib
#9 161.6 checking for special C compiler options needed for large files... no
#9 161.6 checking for _FILE_OFFSET_BITS value needed for large files... no
#9 161.7 checking for special C compiler options needed for large files... (cached) no
#9 161.7 checking for _FILE_OFFSET_BITS value needed for large files... (cached) no
#9 161.7 checking how to print strings... printf
#9 161.7 checking for a sed that does not truncate output... /bin/sed
#9 161.7 checking for fgrep... /bin/grep -F
#9 161.7 checking for ld used by gcc... /usr/x86_64-alpine-linux-musl/bin/ld
#9 161.7 checking if the linker (/usr/x86_64-alpine-linux-musl/bin/ld) is GNU ld... yes
#9 161.7 checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
#9 161.7 checking the name lister (/usr/bin/nm -B) interface... BSD nm
#9 161.7 checking whether ln -s works... yes
#9 161.7 checking the maximum length of command line arguments... 98304
#9 161.7 checking how to convert x86_64-pc-linux-musl file names to x86_64-pc-linux-musl format... func_convert_file_noop
#9 161.7 checking how to convert x86_64-pc-linux-musl file names to toolchain format... func_convert_file_noop
#9 161.7 checking for /usr/x86_64-alpine-linux-musl/bin/ld option to reload object files... -r
#9 161.7 checking for file... no
#9 161.7 checking for objdump... objdump
#9 161.7 checking how to recognize dependent libraries... pass_all
#9 161.7 checking for dlltool... no
#9 161.7 checking how to associate runtime and link libraries... printf %s\n
#9 161.7 checking for archiver @FILE support... @
#9 161.8 checking for strip... strip
#9 161.8 checking for ranlib... (cached) ranlib
#9 161.8 checking command to parse /usr/bin/nm -B output from gcc object... ok
#9 161.9 checking for sysroot... no
#9 161.9 checking for a working dd... /bin/dd
#9 161.9 checking how to truncate binary pipes... /bin/dd bs=4096 count=1
#9 161.9 checking for mt... no
#9 161.9 checking if : is a manifest tool... no
#9 161.9 checking for dlfcn.h... yes
#9 161.9 checking for objdir... .libs
#9 162.0 checking if gcc supports -fno-rtti -fno-exceptions... no
#9 162.0 checking for gcc option to produce PIC... -fPIC -DPIC
#9 162.0 checking if gcc PIC flag -fPIC -DPIC works... yes
#9 162.1 checking if gcc static flag -static works... yes
#9 162.1 checking if gcc supports -c -o file.o... yes
#9 162.1 checking if gcc supports -c -o file.o... (cached) yes
#9 162.1 checking whether the gcc linker (/usr/x86_64-alpine-linux-musl/bin/ld) supports shared libraries... yes
#9 162.2 checking whether -lc should be explicitly linked in... no
#9 162.2 checking dynamic linker characteristics... GNU/Linux ld.so
#9 162.2 checking how to hardcode library paths into programs... immediate
#9 162.2 checking whether stripping libraries is possible... yes
#9 162.3 checking if libtool supports shared libraries... yes
#9 162.3 checking whether to build shared libraries... yes
#9 162.3 checking whether to build static libraries... yes
#9 162.3 checking for bison... bison -y
#9 162.3 checking for flex... flex
#9 162.3 checking for lex output file root... lex.yy
#9 162.3 checking for lex library... none needed
#9 162.7 checking for library containing yywrap... no
#9 162.8 checking whether yytext is a pointer... yes
#9 163.2 checking whether gcc understands -Wall... yes
#9 163.3 checking whether gcc understands -Wformat... yes
#9 163.3 checking whether gcc understands -Wformat-security... yes
#9 163.4 checking whether gcc understands -Wmissing-prototypes... yes
#9 163.4 checking whether gcc understands -Wnested-externs... yes
#9 163.4 checking whether gcc understands -Wpointer-arith... yes
#9 163.5 checking whether gcc understands -Wextra... yes
#9 163.5 checking whether gcc understands -Wshadow... yes
#9 163.6 checking whether gcc understands -Wcast-align... yes
#9 163.6 checking whether gcc understands -Wwrite-strings... yes
#9 163.7 checking whether gcc understands -Waggregate-return... yes
#9 163.7 checking whether gcc understands -Wstrict-prototypes... yes
#9 163.8 checking whether gcc understands -Winline... yes
#9 163.8 checking whether gcc understands -Wredundant-decls... yes
#9 163.8 checking whether gcc understands -Wno-sign-compare... yes
#9 163.9 checking whether gcc understands -fexceptions... yes
#9 163.9 checking whether gcc understands -fasynchronous-unwind-tables... yes
#9 164.0 checking what language compliance flags to pass to the C compiler... 
#9 164.0 checking for readline/readline.h... yes
#9 164.0 checking for readline in -lreadline... yes
#9 164.1 checking for rl_completion_matches... yes
#9 164.1 checking for rl_crlf... yes
#9 164.2 checking for rl_replace_line... yes
#9 164.2 checking for open_memstream... yes
#9 164.3 checking for uselocale... yes
#9 164.3 checking how to pass version script to the linker (/usr/x86_64-alpine-linux-musl/bin/ld)... -Wl,--version-script=
#9 164.3 checking for size_t... yes
#9 164.4 checking for working alloca.h... yes
#9 164.4 checking for alloca... yes
#9 164.4 checking for C/C++ restrict keyword... __restrict__
#9 164.5 checking whether <wchar.h> uses 'inline' correctly... yes
#9 164.5 checking for btowc... yes
#9 164.5 checking for canonicalize_file_name... no
#9 164.6 checking for faccessat... yes
#9 164.6 checking for realpath... yes
#9 164.7 checking for lstat... yes
#9 164.7 checking for _set_invalid_parameter_handler... no
#9 164.8 checking for fcntl... yes
#9 164.8 checking for symlink... yes
#9 164.9 checking for fnmatch... yes
#9 164.9 checking for mbsrtowcs... yes
#9 165.0 checking for getdelim... yes
#9 165.0 checking for getdtablesize... yes
#9 165.0 checking for getrandom... yes
#9 165.1 checking for isblank... yes
#9 165.1 checking for mbsinit... yes
#9 165.2 checking for mbrtowc... yes
#9 165.2 checking for mprotect... yes
#9 165.3 checking for mkstemp... yes
#9 165.3 checking for nl_langinfo... yes
#9 165.4 checking for readlink... yes
#9 165.4 checking for iswctype... yes
#9 165.5 checking for stpncpy... yes
#9 165.5 checking for strndup... yes
#9 165.6 checking for vasnprintf... no
#9 165.6 checking for snprintf... yes
#9 165.7 checking for wcrtomb... yes
#9 165.7 checking for iswcntrl... yes
#9 165.8 checking for wmempcpy... no
#9 165.8 checking for ftruncate... yes
#9 165.9 checking for gettimeofday... yes
#9 165.9 checking for duplocale... yes
#9 165.9 checking for newlocale... yes
#9 166.0 checking for uselocale... (cached) yes
#9 166.0 checking for freelocale... yes
#9 166.0 checking for strerror_r... yes
#9 166.1 checking for __xpg_strerror_r... yes
#9 166.1 checking for pipe... yes
#9 166.2 checking for pthread_sigmask... yes
#9 166.2 checking for setenv... yes
#9 166.3 checking for sigaction... yes
#9 166.3 checking for sigaltstack... yes
#9 166.4 checking for siginterrupt... yes
#9 166.4 checking for sleep... yes
#9 166.5 checking for catgets... yes
#9 166.5 checking for shutdown... yes
#9 166.5 checking for usleep... yes
#9 166.6 checking for wctob... yes
#9 166.6 checking for nl_langinfo and CODESET... yes
#9 166.7 checking for a traditional french locale... none
#9 166.8 checking whether lstat correctly handles trailing slash... yes
#9 166.8 checking whether // is distinct from /... no
#9 166.8 checking whether realpath works... nearly
#9 166.9 checking for getcwd... yes
#9 166.9 checking whether the preprocessor supports include_next... yes
#9 167.0 checking whether source code line length is unlimited... yes
#9 167.0 checking for complete errno.h... yes
#9 167.0 checking for working fcntl.h... yes
#9 168.1 checking for pid_t... yes
#9 168.2 checking for mode_t... yes
#9 168.3 checking for mbstate_t... yes
#9 168.3 checking whether stat file-mode macros are broken... no
#9 168.3 checking for nlink_t... yes
#9 168.4 checking for gcc options needed to detect all undeclared functions... none needed
#9 168.5 checking whether getdelim is declared... yes
#9 168.5 checking whether getdtablesize is declared... yes
#9 168.6 checking whether getline is declared... yes
#9 168.6 checking for getopt.h... (cached) yes
#9 168.6 checking for getopt_long_only... yes
#9 168.7 checking whether getopt is POSIX compatible... no
#9 168.8 checking for pthread.h... (cached) yes
#9 168.9 checking for pthread_kill in -lpthread... yes
#9 168.9 checking whether POSIX threads API is available... yes
#9 169.0 checking whether setlocale (LC_ALL, NULL) is multithread-safe... no
#9 169.0 checking whether setlocale (category, NULL) is multithread-safe... yes
#9 169.0 checking whether imported symbols can be declared weak... yes
#9 169.1 checking whether limits.h has LLONG_MAX, WORD_BIT, ULLONG_WIDTH etc.... no
#9 169.1 checking for wint_t... yes
#9 169.1 checking whether wint_t is large enough... yes
#9 169.1 checking whether the compiler produces multi-arch binaries... no
#9 169.2 checking whether stdint.h conforms to C99... yes
#9 169.3 checking whether stdint.h works without ISO C predefines... yes
#9 169.3 checking whether stdint.h has UINTMAX_WIDTH etc.... no
#9 169.3 checking for wchar_t... yes
#9 169.4 checking for good max_align_t... yes
#9 169.4 checking whether NULL can be used in arbitrary expressions... yes
#9 169.4 checking whether locale.h defines locale_t... yes
#9 169.5 checking whether imported symbols can be declared weak... (cached) yes
#9 169.5 checking for multithread API to use... posix
#9 169.5 checking whether malloc, realloc, calloc are POSIX compliant... yes
#9 169.5 checking for GNU libc compatible malloc... yes
#9 169.5 checking for a traditional japanese locale... none
#9 169.6 checking for a french Unicode locale... none
#9 169.7 checking for a transitional chinese locale... none
#9 169.7 checking for inline... inline
#9 169.7 checking for mmap... yes
#9 169.8 checking for MAP_ANONYMOUS... yes
#9 169.8 checking whether memchr works... yes
#9 169.9 checking whether <limits.h> defines MIN and MAX... no
#9 169.9 checking whether <sys/param.h> defines MIN and MAX... yes
#9 169.9 checking for O_CLOEXEC... yes
#9 170.0 checking for promoted mode_t type... mode_t
#9 170.0 checking whether alarm is declared... yes
#9 170.0 checking for library containing setfilecon... no
#9 170.1 checking for stdbool.h that conforms to C99... yes
#9 170.2 checking for _Bool... yes
#9 170.3 checking whether fcloseall is declared... no
#9 170.3 checking whether ecvt is declared... yes
#9 170.3 checking whether fcvt is declared... yes
#9 170.4 checking whether gcvt is declared... yes
#9 170.4 checking whether stpncpy is declared... yes
#9 170.5 checking whether strndup is declared... yes
#9 170.5 checking whether strnlen is declared... yes
#9 170.6 checking whether strstr works... yes
#9 170.6 checking for struct timespec in <time.h>... yes
#9 170.6 checking for TIME_UTC in <time.h>... yes
#9 170.7 checking whether execvpe is declared... yes
#9 170.7 checking for inttypes.h... yes
#9 170.7 checking for stdint.h... yes
#9 170.8 checking for intmax_t... yes
#9 170.8 checking where to find the exponent in a 'double'... word 1 bit 20
#9 170.9 checking whether snprintf returns a byte count as in C99... yes
#9 170.9 checking whether snprintf truncates the result as in C99... yes
#9 171.0 checking for snprintf... (cached) yes
#9 171.0 checking for strnlen... yes
#9 171.0 checking for wcslen... yes
#9 171.1 checking for wcsnlen... yes
#9 171.1 checking for mbrtowc... (cached) yes
#9 171.1 checking for wcrtomb... (cached) yes
#9 171.1 checking whether _snprintf is declared... no
#9 171.2 checking whether <sys/socket.h> is self-contained... yes
#9 171.2 checking for shutdown... (cached) yes
#9 171.2 checking whether <sys/socket.h> defines the SHUT_* macros... yes
#9 171.2 checking for struct sockaddr_storage... yes
#9 171.3 checking for sa_family_t... yes
#9 171.4 checking for struct sockaddr_storage.ss_family... yes
#9 171.4 checking if environ is properly declared... yes
#9 171.4 checking whether getcwd (NULL, 0) allocates memory for result... yes
#9 171.5 checking for getcwd with POSIX signature... yes
#9 171.5 checking for struct timeval... yes
#9 171.5 checking for wide-enough struct timeval.tv_sec member... yes
#9 171.6 checking for IPv4 sockets... yes
#9 171.6 checking for IPv6 sockets... yes
#9 171.6 checking for LC_MESSAGES... yes
#9 171.7 checking whether uselocale works... yes
#9 171.7 checking for fake locale system (OpenBSD)... no
#9 171.8 checking for Solaris 11.4 locale system... no
#9 171.8 checking for getlocalename_l... no
#9 171.8 checking for CFPreferencesCopyAppValue... no
#9 171.9 checking for CFLocaleCopyPreferredLanguages... no
#9 171.9 checking for library needed for semaphore functions... none
#9 171.9 checking whether <sys/select.h> is self-contained... yes
#9 172.0 checking for library containing setsockopt... none needed
#9 172.1 checking whether select supports a 0 argument... yes
#9 172.1 checking whether select detects invalid fds... yes
#9 172.2 checking whether strerror(0) succeeds... yes
#9 172.2 checking for strerror_r with POSIX signature... yes
#9 172.3 checking whether strerror_r works... yes
#9 172.3 checking whether strerror_r is declared... yes
#9 172.4 checking for sigset_t... yes
#9 172.4 checking whether setenv is declared... yes
#9 172.5 checking for search.h... yes
#9 172.5 checking for tsearch... yes
#9 172.6 checking for uid_t in sys/types.h... yes
#9 172.6 checking whether unsetenv is declared... yes
#9 172.6 checking for alloca as a compiler built-in... yes
#9 172.6 checking for argz.h... no
#9 172.7 checking for error_t... no
#9 172.7 checking for argz_replace... no
#9 172.8 checking whether btowc(0) is correct... yes
#9 172.8 checking whether btowc(EOF) is correct... guessing yes
#9 172.8 checking for __builtin_expect... yes
#9 172.8 checking for library containing clock_gettime... none required
#9 172.9 checking for clock_gettime... yes
#9 172.9 checking for clock_settime... yes
#9 173.0 checking whether // is distinct from /... (cached) no
#9 173.0 checking whether dup2 works... yes
#9 173.0 checking whether fcntl handles F_DUPFD correctly... yes
#9 173.1 checking whether fcntl understands F_DUPFD_CLOEXEC... needs runtime check
#9 173.2 checking for flexible array member... yes
#9 173.2 checking whether conversion from 'int' to 'long double' works... yes
#9 173.3 checking for working POSIX fnmatch... yes
#9 173.3 checking whether free is known to preserve errno... no
#9 173.4 checking for working getdelim function... yes
#9 173.4 checking whether getdtablesize works... yes
#9 173.5 checking for getline... yes
#9 173.5 checking for working getline function... yes
#9 173.6 checking whether getrandom is compatible with its GNU+BSD signature... yes
#9 173.6 checking whether langinfo.h defines CODESET... yes
#9 173.6 checking whether langinfo.h defines T_FMT_AMPM... yes
#9 173.7 checking whether langinfo.h defines ALTMON_1... no
#9 173.7 checking whether langinfo.h defines ERA... yes
#9 173.7 checking whether langinfo.h defines YESEXPR... yes
#9 173.8 checking whether the compiler supports the __inline keyword... yes
#9 173.8 checking whether locale.h conforms to POSIX:2001... yes
#9 173.8 checking whether struct lconv is properly defined... yes
#9 173.9 checking for pthread_rwlock_t... yes
#9 173.9 checking whether pthread_rwlock_rdlock prefers a writer to a reader... no
#9 174.2 checking whether mbrtowc handles incomplete characters... guessing yes
#9 174.2 checking whether mbrtowc works as well as mbtowc... guessing yes
#9 174.2 checking whether mbrtowc handles a NULL pwc argument... guessing yes
#9 174.2 checking whether mbrtowc handles a NULL string argument... guessing yes
#9 174.2 checking whether mbrtowc has a correct return value... guessing yes
#9 174.2 checking whether mbrtowc returns 0 when parsing a NUL character... guessing yes
#9 174.2 checking whether mbrtowc stores incomplete characters... guessing no
#9 174.2 checking whether mbrtowc works on empty input... yes
#9 174.2 checking whether the C locale is free of encoding errors... yes
#9 174.3 checking whether mbrtowc handles incomplete characters... (cached) guessing yes
#9 174.3 checking whether mbrtowc works as well as mbtowc... (cached) guessing yes
#9 174.3 checking whether mbrtowc handles incomplete characters... (cached) guessing yes
#9 174.3 checking whether mbrtowc works as well as mbtowc... (cached) guessing yes
#9 174.3 checking whether mbsrtowcs works... guessing yes
#9 174.3 checking for mbtowc... yes
#9 174.4 checking for mempcpy... yes
#9 174.4 checking whether mkdir handles trailing slash... yes
#9 174.5 checking whether mkdir handles trailing dot... yes
#9 174.5 checking for working mkstemp... yes
#9 174.6 checking whether YESEXPR works... yes
#9 174.7 checking whether open recognizes a trailing slash... yes
#9 174.7 checking for rawmemchr... no
#9 174.8 checking whether readlink signature is correct... yes
#9 174.8 checking whether readlink handles trailing slash correctly... yes
#9 174.9 checking whether readlink truncates results correctly... yes
#9 174.9 checking for working re_compile_pattern... no
#9 175.0 checking for libintl.h... no
#9 175.0 checking whether isblank is declared... yes
#9 175.0 checking whether setlocale (LC_ALL, NULL) is multithread-safe... (cached) no
#9 175.0 checking whether setlocale (category, NULL) is multithread-safe... (cached) yes
#9 175.0 checking whether imported symbols can be declared weak... (cached) yes
#9 175.0 checking whether the -Werror option is usable... yes
#9 175.1 checking for simple visibility declarations... yes
#9 175.1 checking for stdint.h... (cached) yes
#9 175.1 checking for SIZE_MAX... yes
#9 175.1 checking for ssize_t... yes
#9 175.2 checking whether stat handles trailing slashes on files... yes
#9 175.2 checking for struct stat.st_atim.tv_nsec... yes
#9 175.3 checking whether struct stat.st_atim is of type struct timespec... yes
#9 175.3 checking for struct stat.st_birthtimespec.tv_nsec... no
#9 175.3 checking for struct stat.st_birthtimensec... no
#9 175.4 checking for struct stat.st_birthtim.tv_nsec... no
#9 175.5 checking for working stdalign.h... yes
#9 175.5 checking for good max_align_t... (cached) yes
#9 175.5 checking whether NULL can be used in arbitrary expressions... (cached) yes
#9 175.5 checking which flavor of printf attribute matches inttypes macros... system
#9 175.5 checking for stpcpy... yes
#9 175.6 checking for working stpncpy... yes
#9 175.6 checking for strchrnul... yes
#9 175.7 checking whether strchrnul works... yes
#9 175.7 checking for working strndup... yes
#9 175.8 checking for working strnlen... yes
#9 175.9 checking whether strstr works in linear time... yes
#9 175.9 checking whether strstr works... (cached) yes
#9 175.9 checking for nlink_t... (cached) yes
#9 175.9 checking for ptrdiff_t... yes
#9 176.0 checking for vasprintf... yes
#9 176.1 checking whether wcsdup is declared... yes
#9 176.1 checking whether mbrtowc handles incomplete characters... (cached) guessing yes
#9 176.1 checking whether mbrtowc works as well as mbtowc... (cached) guessing yes
#9 176.1 checking whether wcrtomb works in the C locale... yes
#9 176.2 checking whether wcrtomb return value is correct... guessing yes
#9 176.2 checking whether iswcntrl works... yes
#9 176.2 checking for towlower... yes
#9 176.3 checking for wctype_t... yes
#9 176.3 checking for wctrans_t... yes
#9 176.4 checking for wmemchr... yes
#9 176.4 checking for stdint.h... (cached) yes
#9 176.4 checking for a traditional french locale... (cached) none
#9 176.4 checking for a french Unicode locale... (cached) none
#9 176.4 checking for a traditional french locale... (cached) none
#9 176.4 checking for a turkish Unicode locale... none
#9 176.5 checking whether fdopen sets errno... yes
#9 176.5 checking for getpagesize... yes
#9 176.6 checking whether getpagesize is declared... yes
#9 176.6 checking for gettimeofday with POSIX signature... yes
#9 176.7 checking for library containing inet_pton... none required
#9 176.7 checking whether inet_pton is declared... yes
#9 176.7 checking whether byte ordering is bigendian... no
#9 176.9 checking whether INT32_MAX < INTMAX_MAX... yes
#9 176.9 checking whether INT64_MAX == LONG_MAX... yes
#9 176.9 checking whether UINT32_MAX < UINTMAX_MAX... yes
#9 177.0 checking whether UINT64_MAX == ULONG_MAX... yes
#9 177.0 checking for ioctl... yes
#9 177.1 checking for ioctl with POSIX signature... yes
#9 177.1 checking for a traditional french locale... (cached) none
#9 177.1 checking for a french Unicode locale... (cached) none
#9 177.1 checking for a traditional japanese locale... (cached) none
#9 177.1 checking for a transitional chinese locale... (cached) none
#9 177.1 checking for a french Unicode locale... (cached) none
#9 177.1 checking for a traditional french locale... (cached) none
#9 177.1 checking for a french Unicode locale... (cached) none
#9 177.1 checking for a traditional japanese locale... (cached) none
#9 177.1 checking for a transitional chinese locale... (cached) none
#9 177.1 checking for library containing nanosleep... none required
#9 177.1 checking for working nanosleep... no (mishandles large arguments)
#9 178.2 checking whether <netinet/in.h> is self-contained... yes
#9 178.3 checking for a traditional french locale... (cached) none
#9 178.3 checking for a french Unicode locale... (cached) none
#9 178.3 checking whether perror matches strerror... yes
#9 178.3 checking for pthread_t... yes
#9 178.4 checking for pthread_spinlock_t... yes
#9 178.5 checking for PTHREAD_CREATE_DETACHED... yes
#9 178.5 checking for PTHREAD_MUTEX_RECURSIVE... yes
#9 178.6 checking for PTHREAD_MUTEX_ROBUST... yes
#9 178.6 checking for PTHREAD_PROCESS_SHARED... yes
#9 178.7 checking whether pthread_create exists as a global function... yes
#9 178.8 checking whether pthread_sigmask is a macro... no
#9 178.8 checking whether pthread_sigmask works without -lpthread... yes
#9 178.9 checking whether pthread_sigmask returns error numbers... yes
#9 178.9 checking whether pthread_sigmask unblocks signals correctly... guessing yes
#9 181.0 checking for putenv compatible with GNU and SVID... yes
#9 181.1 checking for raise... yes
#9 181.1 checking for sigprocmask... yes
#9 181.2 checking for sched.h... yes
#9 181.2 checking for struct sched_param... yes
#9 181.3 checking whether sched_yield is declared... yes
#9 181.3 checking whether select supports a 0 argument... (cached) yes
#9 181.3 checking whether select detects invalid fds... (cached) yes
#9 181.3 checking whether setenv validates arguments... yes
#9 181.4 checking whether setlocale supports the C locale... yes
#9 181.4 checking for a traditional french locale... (cached) none
#9 181.4 checking for a french Unicode locale... (cached) none
#9 181.4 checking for a traditional japanese locale... (cached) none
#9 181.4 checking for a transitional chinese locale... (cached) none
#9 181.4 checking for struct sigaction.sa_sigaction... yes
#9 181.5 checking for volatile sig_atomic_t... yes
#9 181.6 checking for sighandler_t... yes
#9 181.7 checking for sigprocmask... (cached) yes
#9 181.7 checking whether sleep is declared... yes
#9 181.7 checking for working sleep... yes
#9 182.8 checking for socklen_t... yes
#9 182.8 checking for working strerror function... yes
#9 182.9 checking whether symlink handles trailing slash correctly... yes
#9 182.9 checking whether <sys/ioctl.h> declares ioctl... yes
#9 183.0 checking for pthread_atfork... yes
#9 183.0 checking for sys/single_threaded.h... no
#9 183.1 checking for unsetenv... yes
#9 183.1 checking for unsetenv() return type... int
#9 183.2 checking whether unsetenv obeys POSIX... yes
#9 183.2 checking for useconds_t... yes
#9 183.3 checking whether usleep allows large arguments... yes
#9 184.4 checking for a traditional french locale... (cached) none
#9 184.4 checking for a french Unicode locale... (cached) none
#9 184.4 checking for a traditional japanese locale... (cached) none
#9 184.4 checking for a transitional chinese locale... (cached) none
#9 184.4 checking whether wctob works... guessing yes
#9 184.4 checking whether wctob is declared... yes
#9 184.4 checking if we should run the GNUlib tests... yes
#9 184.4 checking for pkg-config... /usr/bin/pkg-config
#9 184.4 checking pkg-config is at least version 0.9.0... yes
#9 184.4 checking for LIBXML... yes
#9 184.4 checking for strerror_r... (cached) yes
#9 184.4 checking for fsync... yes
#9 184.5 checking that generated files are newer than configure... done
#9 184.5 configure: creating ./config.status
#9 184.9 config.status: creating Makefile
#9 185.0 config.status: creating gnulib/lib/Makefile
#9 185.1 config.status: creating gnulib/tests/Makefile
#9 185.2 config.status: creating src/Makefile
#9 185.2 config.status: creating man/Makefile
#9 185.3 config.status: creating tests/Makefile
#9 185.4 config.status: creating examples/Makefile
#9 185.4 config.status: creating doc/Makefile
#9 185.5 config.status: creating doc/naturaldocs/Makefile
#9 185.5 config.status: creating augeas.pc
#9 185.5 config.status: creating augeas.spec
#9 185.6 config.status: creating config.h
#9 185.6 config.status: executing depfiles commands
#9 186.7 config.status: executing libtool commands
#9 186.8 
#9 186.8 Now type 'make' to compile augeas.
#9 186.8 + make
#9 186.8 make  all-recursive
#9 186.8 make[1]: Entering directory '/augeas'
#9 186.8 Making all in gnulib/lib
#9 186.9 make[2]: Entering directory '/augeas/gnulib/lib'
#9 186.9   GEN      alloca.h
#9 186.9   GEN      argz.h
#9 186.9   GEN      ctype.h
#9 186.9   GEN      fcntl.h
#9 186.9   GEN      getopt.h
#9 186.9   GEN      getopt-cdefs.h
#9 186.9   GEN      inttypes.h
#9 186.9   GEN      langinfo.h
#9 187.0   GEN      limits.h
#9 187.0   GEN      locale.h
#9 187.0   GEN      selinux/selinux.h
#9 187.0   GEN      selinux/context.h
#9 187.0   GEN      selinux/label.h
#9 187.0   GEN      stdint.h
#9 187.1   GEN      stdio.h
#9 187.2   GEN      stdlib.h
#9 187.3   GEN      string.h
#9 187.4   GEN      sys/random.h
#9 187.4   GEN      sys/stat.h
#9 187.4   GEN      sys/types.h
#9 187.4   GEN      sys/wait.h
#9 187.5   GEN      time.h
#9 187.5   GEN      unistd.h
#9 187.6   GEN      wchar.h
#9 187.7   GEN      wctype.h
#9 187.7 make  all-recursive
#9 187.7 make[3]: Entering directory '/augeas/gnulib/lib'
#9 187.8 make[4]: Entering directory '/augeas/gnulib/lib'
#9 187.8   CC       basename-lgpl.lo
#9 187.9   CC       cloexec.lo
#9 188.0   CC       dirname-lgpl.lo
#9 188.1   CC       stripslash.lo
#9 188.3   CC       malloc/dynarray_at_failure.lo
#9 188.4   CC       malloc/dynarray_emplace_enlarge.lo
#9 188.6   CC       malloc/dynarray_finalize.lo
#9 188.7   CC       malloc/dynarray_resize.lo
#9 188.9   CC       malloc/dynarray_resize_clear.lo
#9 189.0   CC       fd-hook.lo
#9 189.1   CC       hard-locale.lo
#9 189.3   CC       localcharset.lo
#9 189.4   CC       glthread/lock.lo
#9 189.7   CC       malloca.lo
#9 189.8   CC       safe-alloc.lo
#9 190.0   CC       malloc/scratch_buffer_dupfree.lo
#9 190.2   CC       malloc/scratch_buffer_grow.lo
#9 190.3   CC       malloc/scratch_buffer_grow_preserve.lo
#9 190.5   CC       malloc/scratch_buffer_set_array_size.lo
#9 190.6   CC       se-context.lo
#9 190.7   CC       se-label.lo
#9 190.9   CC       se-selinux.lo
#9 191.0   CC       setlocale_null.lo
#9 191.2   CC       stat-time.lo
#9 191.4   CC       strnlen1.lo
#9 191.5   CC       tempname.lo
#9 191.7   CC       glthread/threadlib.lo
#9 191.8   CC       unistd.lo
#9 191.9   CC       wctype-h.lo
#9 192.0   CC       xsize.lo
#9 192.1   CC       argz.lo
#9 192.6   CC       asnprintf.lo
#9 192.7   CC       canonicalize-lgpl.lo
#9 192.8 canonicalize-lgpl.c: In function 'rpl_realpath':
#9 192.8 canonicalize-lgpl.c:434:5: warning: #warning "GCC might issue a bogus -Wreturn-local-addr warning here." [-Wcpp]
#9 192.8   434 |    #warning "GCC might issue a bogus -Wreturn-local-addr warning here."
#9 192.8       |     ^~~~~~~
#9 192.8 canonicalize-lgpl.c:435:5: warning: #warning "See <https://gcc.gnu.org/bugzilla/show_bug.cgi?id=93644>." [-Wcpp]
#9 192.8   435 |    #warning "See <https://gcc.gnu.org/bugzilla/show_bug.cgi?id=93644>."
#9 192.8       |     ^~~~~~~
#9 193.1   CC       fcntl.lo
#9 193.3   CC       free.lo
#9 193.4   CC       getopt.lo
#9 194.0   CC       getopt1.lo
#9 194.1   CC       nl_langinfo.lo
#9 194.3   CC       printf-args.lo
#9 194.5   CC       printf-parse.lo
#9 195.3   CC       rawmemchr.lo
#9 195.4   CC       regex.lo
#9 204.4   CC       setlocale-lock.lo
#9 204.5   CC       vasnprintf.lo
#9 205.3   CC       wmempcpy.lo
#9 205.4   CCLD     libgnu.la
#9 205.7 make[4]: Leaving directory '/augeas/gnulib/lib'
#9 205.7 make[3]: Leaving directory '/augeas/gnulib/lib'
#9 205.7 make[2]: Leaving directory '/augeas/gnulib/lib'
#9 205.7 Making all in src
#9 205.7 make[2]: Entering directory '/augeas/src'
#9 205.7 echo '#define DATADIR "/opt/augeas/share"' > datadir.h1
#9 205.7 ../build/ac-aux/move-if-change datadir.h1 datadir.h
#9 205.7 make  all-am
#9 205.7 make[3]: Entering directory '/augeas/src'
#9 205.7   CC       augtool.o
#9 206.1   CC       augeas.lo
#9 208.6   CC       augrun.lo
#9 208.8 augrun.c: In function 'nexttoken':
#9 208.8 augrun.c:160:24: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 208.8   160 |                     if (quot) break;
#9 208.8       |                        ^
#9 208.8 augrun.c:162:17: note: here
#9 208.8   162 |                 case '\'':
#9 208.8       |                 ^~~~
#9 208.8 augrun.c:164:24: warning: this statement may fall through [-Wimplicit-fallthrough=]
#9 208.8   164 |                     if (quot && quot != *(s+1)) break;
#9 208.8       |                        ^
#9 208.8 augrun.c:166:17: note: here
#9 208.8   166 |                 case '\\': /* pass next character through */
#9 208.8       |                 ^~~~
#9 210.1   CC       pathx.lo
#9 213.6   CC       internal.lo
#9 213.7 internal.c: In function 'xstrerror':
#9 213.7 internal.c:436:12: warning: returning 'int' from a function with return type 'const char *' makes pointer from integer without a cast [-Wint-conversion]
#9 213.7   436 |     return strerror_r(errnum, buf, len);
#9 213.7       |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
#9 214.3   CC       memory.lo
#9 214.6   CC       ref.lo
#9 214.7   CC       syntax.lo
#9 214.8 syntax.c: In function 'syntax_error':
#9 214.8 syntax.c:134:5: warning: this 'if' clause does not guard... [-Wmisleading-indentation]
#9 214.8   134 |     if (error->code != AUG_NOERROR && error->code != AUG_ESYNTAX)
#9 214.8       |     ^~
#9 214.8 In file included from syntax.c:26:
#9 214.8 syntax.c:137:9: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the 'if'
#9 214.8   137 |         va_start(ap, format);
#9 214.8       |         ^~~~~~~~
#9 214.8 syntax.c: In function 'fatal_error':
#9 214.8 syntax.c:146:5: warning: this 'if' clause does not guard... [-Wmisleading-indentation]
#9 214.8   146 |     if (error->code == AUG_EINTERNAL)
#9 214.8       |     ^~
#9 214.8 In file included from syntax.c:26:
#9 214.8 syntax.c:149:9: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the 'if'
#9 214.8   149 |         va_start(ap, format);
#9 214.8       |         ^~~~~~~~
#9 217.8   YACC     parser.c
#9 217.8 /augeas/src/parser.y:46.1-5: warning: POSIX Yacc does not support %code [-Wyacc]
#9 217.8    46 | %code provides {
#9 217.8       | ^~~~~
#9 217.8 /augeas/src/parser.y:58.1-14: warning: POSIX Yacc does not support %error-verbose [-Wyacc]
#9 217.8    58 | %error-verbose
#9 217.8       | ^~~~~~~~~~~~~~
#9 217.9 /augeas/src/parser.y:58.1-14: warning: deprecated directive: ‘%error-verbose’, use ‘%define parse.error verbose’ [-Wdeprecated]
#9 217.9    58 | %error-verbose
#9 217.9       | ^~~~~~~~~~~~~~
#9 217.9       | %define parse.error verbose
#9 217.9 /augeas/src/parser.y:59.14-20: warning: POSIX Yacc does not support string literals [-Wyacc]
#9 217.9    59 | %name-prefix "augl_"
#9 217.9       |              ^~~~~~~
#9 217.9 /augeas/src/parser.y:59.1-20: warning: POSIX Yacc does not support %name-prefix  [-Wyacc]
#9 217.9    59 | %name-prefix "augl_"
#9 217.9       | ^~~~~~~~~~~~~~~~~~~~
#9 217.9 /augeas/src/parser.y:59.1-20: warning: deprecated directive: ‘%name-prefix "augl_"’, use ‘%define api.prefix {augl_}’ [-Wdeprecated]
#9 217.9    59 | %name-prefix "augl_"
#9 217.9       | ^~~~~~~~~~~~~~~~~~~~
#9 217.9       | %define api.prefix {augl_}
#9 217.9 /augeas/src/parser.y:60.1-8: warning: POSIX Yacc does not support %defines [-Wyacc]
#9 217.9    60 | %defines
#9 217.9       | ^~~~~~~~
#9 217.9 /augeas/src/parser.y:61.1-12: warning: POSIX Yacc does not support %pure-parser [-Wyacc]
#9 217.9    61 | %pure-parser
#9 217.9       | ^~~~~~~~~~~~
#9 217.9 /augeas/src/parser.y:61.1-12: warning: deprecated directive: ‘%pure-parser’, use ‘%define api.pure’ [-Wdeprecated]
#9 217.9    61 | %pure-parser
#9 217.9       | ^~~~~~~~~~~~
#9 217.9       | %define api.pure
#9 217.9 /augeas/src/parser.y:66.1-15: warning: POSIX Yacc does not support %initial-action [-Wyacc]
#9 217.9    66 | %initial-action {
#9 217.9       | ^~~~~~~~~~~~~~~
#9 218.1 /augeas/src/parser.y: warning: fix-its can be applied.  Rerun with option '--update'. [-Wother]
#9 218.1 updating parser.h
#9 218.1   CC       parser.lo
#9 219.7   CC       builtin.lo
#9 220.8   CC       lens.lo
#9 224.0   CC       regexp.lo
#9 224.9   CC       transform.lo
#9 225.1 transform.c: In function 'transfer_file_attrs':
#9 225.1 transform.c:921:5: error: unknown type name 'security_context_t'; did you mean 'security_class_t'?
#9 225.1   921 |     security_context_t con = NULL;
#9 225.1       |     ^~~~~~~~~~~~~~~~~~
#9 225.1       |     security_class_t
#9 225.1 transform.c:921:30: warning: initialization of 'int' from 'void *' makes integer from pointer without a cast [-Wint-conversion]
#9 225.1   921 |     security_context_t con = NULL;
#9 225.1       |                              ^~~~
#9 225.1 transform.c:939:34: warning: passing argument 2 of 'fgetfilecon' from incompatible pointer type [-Wincompatible-pointer-types]
#9 225.1   939 |         if (fgetfilecon(from_fd, &con) < 0 && errno != ENOTSUP) {
#9 225.1       |                                  ^~~~
#9 225.1       |                                  |
#9 225.1       |                                  int *
#9 225.1 In file included from transform.c:32:
#9 225.1 ../gnulib/lib/selinux/selinux.h:108:29: note: expected 'char **' but argument is of type 'int *'
#9 225.1   108 | fgetfilecon (int fd, char **con _GL_UNUSED_PARAMETER)
#9 225.1 transform.c:953:32: warning: comparison between pointer and integer
#9 225.1   953 |     if (selinux_enabled && con != NULL) {
#9 225.1       |                                ^~
#9 225.1 transform.c:954:32: warning: passing argument 2 of 'fsetfilecon' makes pointer from integer without a cast [-Wint-conversion]
#9 225.1   954 |         if (fsetfilecon(to_fd, con) < 0 && errno != ENOTSUP) {
#9 225.1       |                                ^~~
#9 225.1       |                                |
#9 225.1       |                                int
#9 225.1 In file included from transform.c:32:
#9 225.1 ../gnulib/lib/selinux/selinux.h:120:26: note: expected 'const char *' but argument is of type 'int'
#9 225.1   120 |              char const *con _GL_UNUSED_PARAMETER)
#9 225.1 transform.c:958:17: warning: passing argument 1 of 'freecon' makes pointer from integer without a cast [-Wint-conversion]
#9 225.1   958 |         freecon(con);
#9 225.1       |                 ^~~
#9 225.1       |                 |
#9 225.1       |                 int
#9 225.1 In file included from transform.c:32:
#9 225.1 ../gnulib/lib/selinux/selinux.h:86:16: note: expected 'char *' but argument is of type 'int'
#9 225.1    86 | freecon (char *con _GL_UNUSED_PARAMETER) {}
#9 225.1 make[3]: *** [Makefile:1938: transform.lo] Error 1
#9 225.1 make[3]: Leaving directory '/augeas/src'
#9 225.1 make[2]: Leaving directory '/augeas/src'
#9 225.1 make[2]: *** [Makefile:1731: all] Error 2
#9 225.1 make[1]: *** [Makefile:1807: all-recursive] Error 1
#9 225.1 make[1]: Leaving directory '/augeas'
#9 225.1 make: *** [Makefile:1672: all] Error 2
------
executor failed running [/bin/sh -c set -ex ;     cd augeas  && ./autogen.sh --prefix=/opt/augeas  && make  && make install]: exit code: 2
```
- my fix: I git checked out the last Dockerfile commit , assuming the Dockerfile works properly when it's merged.
```bash
 git checkout fa315ce5a59b01f0a3f7730a70857f009f5db2d9
```

However, that commit misses several non-trivial features I"m looking for [compare](https://github.com/hercules-team/augeas/compare/fa315ce5a59b01f0a3f7730a70857f009f5db2d9...master) 

Do we have any plans to fix the docker path?

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -1,6 +1,6 @@
 FROM alpine AS build
 
-RUN apk add --no-cache --virtual=.build gcc libc-dev make bison flex readline-dev libxml2-dev git automake autoconf libtool pkgconf coreutils
+RUN apk add --no-cache --virtual=.build gcc libc-dev make bison flex readline-dev libxml2-dev git automake autoconf libtool pkgconf coreutils bash
 COPY . /augeas
 RUN set -ex ; \
     cd augeas \

